lib/bolt/cli/src/commands/admin/mod.rs

```diff
@@ -0,0 +1,64 @@
+use anyhow::*;
+use bolt_core::{
+	context::ProjectContext,
+	tasks::{self},
+	utils,
+};
+use clap::Parser;
+use rivet_api::apis::*;
+
+mod cluster;
+
+#[derive(Parser)]
+pub enum SubCommand {
+	/// Generates a login link for the given access token. Automatically turns the existing user into an
+	/// admin (or creates a new admin if no user).
+	Login {
+		#[clap(default_value = "root")]
+		name: String,
+	},
+	/// Cluster related operations
+	Cluster {
+		/// The name ID of the cluster to be used. `None` if the cluster is
+		/// going to be created.
+		#[clap(index = 1)]
+		cluster_name_id: Option<String>,
```

Comment:
this should be a subparameter of each subcommand. e.g.

```rust
enum command {
    create {

    },
    delete {
        name_id: string
    },
    taint {
        name_id: string
    }
}
```

Reply:
Max and I were discussing a command layout like:
`bolt admin cluster <CLUSTER_NAME_ID> datacenter <DATACENTER_NAME_ID> delete <extra params>`.

I assumed it would be best to have name ids as arguments, but it does seem like it could make more sense to keep datacenter input as parameters.